backend_with_db.js

```diff
@@ -19,6 +23,19 @@ function generateAccessToken(username) {
   });
 }
 
+// Middleware to validate request body for signup
+const validateSignup = [
+  body('username').isString().withMessage('Username must be a string').trim().escape(),
+  body('pwd').isLength({ min: 6 }).withMessage('Password must be at least 6 characters long'),
```

Comment:
Ups the security, is doing to make fake accounts more of a paint to make. goodbye 123, hello to 123456!